staging/kos/cmd/connection-agent/main.go

```diff
@@ -93,24 +96,24 @@ func main() {
 		// fall back to default node name
 		nodeName, err = os.Hostname()
 		if err != nil {
-			glog.Errorf("-nodename flag value was not provided and default value could not be retrieved: %s\n", err.Error())
+			klog.Errorf("-nodename flag value was not provided and default value could not be retrieved: %s\n", err.Error())
```

Comment:
Thanks for handling the migration from `glog` to `klog`, it's definitely something the community would request in order to accept `kos` as an official example. A considerable amount of our log statements end with a newline, while others don't as it is automatically appended. Can you fix that in this PR by removing the newlines which are still there?

Reply:
done.

Reply:
Thanks!